paper.md

```diff
@@ -0,0 +1,257 @@
+The HDF5group have provided good documentation for the core features of HDF5 which include all those of interest to the weather and climate community who motivated this reboot of pyfive, and while there is a community of developers beyond the HDF5 group (including some at the publicly funded Unidata institution), recent events suggest that given most of those developers and their existing funding are US based, some spreading of risk would be desirable. 
+To that end, a pure python code, which is relatively small and maintained by an international constituency, alongside the existing c-code, provides some assurance that the community can maintain HDF5 access for the foreseeable future. 
+A pure python code also makes it easier to develop scripts which can work around data and metadata damage should they occur. 
+
+## Examples
+
+We now introduce three aspects of the new functionality that `pyfive` now provides: remote access, configurable lazy loading, and determining whether files are cloud optimised.
+
+##  Remote Access
+
+A notable feature of the recent `pyfive` upgrade is that it was carried out with thread-safety and remote access using fsspec (filesystem-spec.readthedocs.io) in mind.  We provide two examples of using `pyfive` to access remote data, one in S3, and one behind a modern http web server:
+
+For accessing the data on S3 storage, we will have to set up an ``s3fs`` virtual file system, then pass it to Pyfive:
+
+```python
+import pyfive
+import s3fs
+# storage options for an anon S3 bucket
+# there are also caching options for the s3 middleware, not shown here
+storage_options = {
+    'anon': True,
+    'client_kwargs': {'endpoint_url': "https://s3server.ac.uk"}
+}
+fs = s3fs.S3FileSystem(**storage_options)
+file_uri = "s3-bucket/myfile.nc"
+with fs.open(file_uri, 'rb') as s3_file:
+    nc = pyfive.File(s3_file)
+    dataset = nc[var]
+```
+
+for an HTTPS data server, the usage is similar:
+
+```python
+import fsspec
+import pyfive
+# there are also caching options for the fsspec middleware, not shown here
+fs = fsspec.filesystem('http')
+with fs.open("https://site.com/myfile.nc", 'rb') as http_file:
+    nc = pyfive.File(http_file)
+    dataset = nc[var]
+```
+
+This is of course exactly the same pattern as remote access using `h5py`, and that is by design - to make
+moving to `pyfive` easy for users!
+
+## Lazy Loading
+
+A key tenet of efficient remote access is that variable inspection is quick and involves the minimimum of network
+traffic between storage and the client application.  
+However when this is coupled with the common pattern of 
+using Dask, some flexibility in what is loaded when is beneficial.
+
+By default when one inspects the contents of a file using `pyfive v1.0` nothing more is read from the file than
```

Comment:
Not sure we need to mention the version, here. I think we can take is as read that we're describing the latest and greatest.

```suggestion
By default when one inspects the contents of a file using `pyfive` nothing more is read from the file than
```